python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -133,9 +133,10 @@ def runDataRef(self, dataRef):
         dataRef : `lsst.daf.persistence.ButlerDataRef`
             Data reference for exposure.
         """
+        self.log.info(f"Processing {dataRef.dataId}")
```

Comment:
For log messages, please use the `%` form of formatting, but only provide the formatting arguments rather than doing the formatting yourself. E.g.,

    self.log.info("Processing %s", dataRef.dataId)

This allows the formatting to be skipped if the log message wouldn't be printed. It's not a big deal here, since you're not in a tight loop, but it can matter in some contexts.